templates/layouts/aw-install-simg.hbs

```diff
@@ -12,7 +12,7 @@
     {{!-- Content --}}
     <div class="container docs-container">
       <h1>Hardware Support</h1>
-      <p>Before installing AsteroidOS to your watch, make sure you are aware of the capabilities and limitations of AsteroidOS on the {{title}}. The following table should summarize the current support of this watch:</p>
+      <p>The following table is a summary of the currently supported features of the {{title}}.</p>
```

Comment:
I think the text is not quite right.  It's not just currently supported features, but also unsupported features that are listed.  Maybe better would be "The following table summarizes the state of the features of {{title}}."

Reply:
Great! That's why i appreciate the review. The sentence also sounded a bit wooden.